src/sinks/mod.rs

```diff
@@ -133,200 +133,249 @@ pub enum HealthcheckError {
 
-    /// GCP Cloud Storage.
+    /// Store observability events in GCP Cloud Storage.
     #[cfg(feature = "sinks-gcp")]
+    #[configurable(metadata(docs::label = "GCP Cloud Storage"))]
     GcpCloudStorage(gcp::cloud_storage::GcsSinkConfig),
 
-    /// GCP Pub/Sub.
+    /// Publish observability events to GCP's Pub/Sub messaging system.
     #[cfg(feature = "sinks-gcp")]
+    #[configurable(metadata(docs::label = "GCP Pub/Sub"))]
     GcpPubsub(gcp::pubsub::PubsubConfig),
 
-    /// Honeycomb.
+    /// Deliver log events to Honeycomb.
     #[cfg(feature = "sinks-honeycomb")]
+    #[configurable(metadata(docs::label = "Honeycomb"))]
     Honeycomb(honeycomb::HoneycombConfig),
 
-    /// HTTP.
+    /// Deliver observability event data to an HTTP server.
     #[cfg(feature = "sinks-http")]
+    #[configurable(metadata(docs::label = "HTTP"))]
     Http(http::HttpSinkConfig),
 
-    /// Humio Logs.
+    /// Deliver log event data to Humio.
     #[cfg(feature = "sinks-humio")]
+    #[configurable(metadata(docs::label = "Humio Logs"))]
     HumioLogs(humio::logs::HumioLogsConfig),
 
-    /// Humio Metrics.
+    /// Deliver metric event data to Humio.
     #[cfg(feature = "sinks-humio")]
+    #[configurable(metadata(docs::label = "Humio Metrics"))]
     HumioMetrics(humio::metrics::HumioMetricsConfig),
 
-    /// InfluxDB Logs.
+    /// Deliver log event data to InfluxDB.
     #[cfg(any(feature = "sinks-influxdb", feature = "prometheus-integration-tests"))]
+    #[configurable(metadata(docs::label = "InfluxDB Logs"))]
     InfluxdbLogs(influxdb::logs::InfluxDbLogsConfig),
 
-    /// InfluxDB Metrics.
+    /// Deliver metric event data to InfluxDB.
     #[cfg(any(feature = "sinks-influxdb", feature = "prometheus-integration-tests"))]
+    #[configurable(metadata(docs::label = "InfluxDB Metrics"))]
     InfluxdbMetrics(influxdb::metrics::InfluxDbConfig),
 
-    /// Kafka.
+    /// Publish observability event data to Apache Kafka topics.
     #[cfg(feature = "sinks-kafka")]
+    #[configurable(metadata(docs::label = "Kafka"))]
     Kafka(kafka::KafkaSinkConfig),
 
-    /// LogDNA.
+    /// Deliver log event data to LogDNA.
     #[cfg(feature = "sinks-logdna")]
+    #[configurable(metadata(docs::label = "LogDNA"))]
     Logdna(logdna::LogdnaConfig),
 
-    /// Loki.
+    /// Deliver log event data to the Loki aggregation system.
     #[cfg(feature = "sinks-loki")]
+    #[configurable(metadata(docs::label = "Loki"))]
     Loki(loki::LokiConfig),
 
-    /// NATS.
+    /// Publish observability data to subjects on the NATS messaging system.
     #[cfg(feature = "sinks-nats")]
+    #[configurable(metadata(docs::label = "NATS"))]
     Nats(self::nats::NatsSinkConfig),
 
-    /// New Relic.
+    /// Deliver events to New Relic.
     #[cfg(feature = "sinks-new_relic")]
+    #[configurable(metadata(docs::label = "New Relic"))]
     NewRelic(new_relic::NewRelicConfig),
 
-    /// Papertrail.
+    /// Deliver log events to Papertrail from SolarWinds.
```

Comment:
**nit (non-blocking)**: Most of the descriptions were in the <company> <product> form, this one seems like an outlier.